pkg/virtual/apiexport/controllers/apireconciler/apiexport_apireconciler_reconcile.go

```diff
@@ -186,15 +197,19 @@ func (c *APIReconciler) reconcile(ctx context.Context, apiExport *apisv1alpha1.A
 				if err != nil {
 					return fmt.Errorf(fmt.Sprintf("failed to convert permission claim %v to label key and value: %v", c, err))
 				}
-				selector := labels.SelectorFromSet(labels.Set{key: label})
-				var selectable bool
-				labelReqs, selectable = selector.Requirements()
-				if !selectable {
-					return fmt.Errorf("permission claim %v for APIExport %s|%s is not selectable", c, clusterName, apiExport.Name)
+				claimLabels := []string{label}
+				if gvr.GroupResource() == apisv1alpha1.Resource("apibindings") {
+					_, fallbackLabel := permissionclaims.ToReflexiveAPIBindingLabelKeyAndValue(logicalcluster.From(apiExport), apiExport.Name)
+					claimLabels = append(claimLabels, fallbackLabel)
+				}
+				req, err := labels.NewRequirement(key, selection.In, claimLabels)
```

Comment:
Does this have a distinction on the server side and any sort of thing we should worry about?

I don't know the inner workings as well as y'all, but I could see check that something has some equal vs. in a list of things could have a performance burden. I mean, the other thing is that we know it is only two things, so maybe it is a moot point, but I did want to bring it up.

Reply:
it's a valid label selector. And label selectors are implemented in memory on the server side. So not noticable difference. Either is slow (we don't have an index afaik).